src/Microsoft.PowerApps.TestEngine/ITestEngineConsoleEvents.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using Microsoft.Extensions.Logging;
+using Microsoft.PowerApps.TestEngine.Config;
+
+namespace Microsoft.PowerApps.TestEngine
+{
+    /// <summary>
+    /// Interface that handles all console related events
+    /// </summary>
+    public interface ITestEngineEvents
```

Comment:
Recommend you match the name of your file to the name of the interface/class in that file, meaning I'd recommend renaming the file in this case.